packages/ndla-ui/src/TagSelector/ValueButton.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2022-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React from 'react';
+import { MultiValueProps } from 'react-select';
+import { buttonStyleV2 as buttonStyle } from '@ndla/button';
+import styled from '@emotion/styled';
+import { colors, spacing } from '@ndla/core';
+import { Cross } from '@ndla/icons/action';
+import { TagType } from './types';
+
+interface StyledProps {
+  selected: boolean;
+}
+
+export const StyledValueButton = styled.div<StyledProps>`
+  && {
+    background: ${({ selected }) => selected && colors.brand.primary};
+    color: ${({ selected }) => selected && colors.white};
+    padding: ${spacing.xxsmall} ${spacing.small};
+    margin: ${spacing.xxsmall};
+    border: none;
+  }
+`;
+
+const ValueButton = ({ innerProps, children, removeProps, isFocused }: MultiValueProps<TagType, true>) => {
+  return (
+    <StyledValueButton
+      selected={isFocused}
+      role="button"
+      css={buttonStyle({ colorTheme: 'lighter', shape: 'pill', size: 'small' })}
+      {...innerProps}
+      {...removeProps}>
+      #{children}
```

Comment:
Skal det være # hardkoda her?

Reply:
Det er en prefix som legges på før tagnavnet.
<img width="82" alt="image" src="https://user-images.githubusercontent.com/17144211/193226431-0b0e8616-914a-4ad9-b32c-97a1c84324c8.png">

Reply:
Skjermleseren vil nok lese opp dette. Burde nok være et ikon

Reply:
Det er jo vanlig at # brukes for emneknagger, men siden vi viser dei i pills, er det egentlig nødvendig å ha med # her?

Reply:
Satte den i en aria-hidden span nå. Den er i designet, så må nesten være med